nexus_constructor/model/load_from_json.py

```diff
@@ -0,0 +1,118 @@
+import json
+import logging
+
+from PySide2.QtWidgets import QWidget
+
+from nexus_constructor.component.component_type import COMPONENT_TYPES
+from nexus_constructor.model.component import Component
+from nexus_constructor.model.entry import Entry
+from nexus_constructor.model.instrument import Instrument
+from nexus_constructor.ui_utils import show_warning_dialog
+
+IGNORE = ["entry", "instrument", "transformations", "NX_class"]
+
+
+def _read_nx_class(entry: list) -> str:
+    for item in entry:
+        if item.get("name") == "NX_class":
+            return item.get("values")
+        if item.get("NX_class"):
+            return item.get("NX_class")
+
+    return ""
+
+
+def _read_transformations(entry: list):
+    for item in entry:
+        if item.get("name") == "transformations":
```

Comment:
I think this is good enough for the initial implementation, but just so that you are aware:
This will work for JSON output from the NeXus Constructor but there is no reason in the NeXus format that this has to be called "transformations".